card/mod/01_core/format/html_format.rb

```diff
@@ -45,15 +40,21 @@ def layout_from_card_or_code name
       end
     end
     
-    
     def get_inclusion_defaults
-      INCLUSION_DEFAULTS[@mode] || {}
+      {}
     end
   
     def default_item_view
       :closed
     end
 
+    def nest nested_card, opts={}
+      unless opts[:view].present?
+        opts[:view] = nested_card.rule :default_html_view
+      end
+      super nested_card, opts
+    end
```

Comment:
This means default_html_view overrides the view defined explicitly in the inclusion (because this is handled in super if there is no view in opts). I don't think that's a good idea.